wagtailio/static/sass/components/_comparison-table.scss

```diff
@@ -7,6 +7,7 @@
 
     td:first-child {
         font-weight: $weight--bold;
+        min-width: fit-content;
```

Comment:
`min-width: fit-content;` doesn’t have any affect for me. I’d be tempted to set a fixed width of `width: 250px` at the large breakpoint and `width: 150px` on the small breakpoint.

Reply:
It would depend on the size of your images in the header and if they pushed the line passed the point of the min-width of the other I guess. Will use your suggestion.